src/sparseml/core/logger.py

```diff
@@ -578,7 +579,7 @@ def __init__(
 
         if os.environ.get("NM_TEST_MODE"):
             test_log_path = os.environ.get("NM_TEST_LOG_DIR")
-            create_dirs(test_log_path)
+            _create_dirs(test_log_path)
```

Comment:
what happened to the original `create_dirs` function? Is it still be used elsewhere?